src/commcare_cloud/environment/main.py

```diff
@@ -105,9 +105,16 @@ def proxy_config(self):
 
     @memoized_property
     def users_config(self):
-        with open(self.paths.get_users_yml(self.meta_config.users)) as f:
-            users_json = yaml.load(f)
-        return UsersConfig.wrap(users_json)
+        user_groups_from_yml = self.meta_config.users
+        absent_users = []
+        present_users = []
+        for user_group_from_yml in user_groups_from_yml:
+            with open(self.paths.get_users_yml(user_group_from_yml)) as f:
+                user_group_json = yaml.load(f)
+                present_users += user_group_json['dev_users']['present']
+                absent_users += user_group_json['dev_users']['absent']
```

Comment:
do the QA & g2 users need to be added here?

Reply:
Implicit in my interpretation of your question is the idea that "qa users" and "g2 users" are a different type of user. You can tell me if that's not how you meant it, but I'll address that point and say that there's only one type of user our ansible is aware of, and it calls all of them "dev_users". The breakdown into `dimagi`, `dimagi-qa`, and `g2` is for our own ability to organize which user accounts and ssh keys get added to (or removed from) which environments. For example, staging gets `dimagi`, `dimagi-qa`, `g2`; production gets `dimagi` and `g2`; most of our custom envs get just `dimagi`.

Reply:
Ah. yeah that makes sense. somehow skipped over the extra users varialbe in environments. Thanks